app/src/main/java/org/asteroidos/sync/connectivity/IService.java

```diff
@@ -0,0 +1,34 @@
+package org.asteroidos.sync.connectivity;
+
+/**
+ * Every Service of Sync has to implement the {@link IService} interface.
+ * A service is a module that can be loaded ({@link IService#sync()}) after the connection is established and
+ * will get destroyed ({@link IService#unsync()}) when the connection is terminated.
+ */
+public interface IService {
+
+    /**
+     * <code>sync()</code> is called on service creation
```

Comment:
This description doesn't match what is said above. "service creation" and "before a service is destroyed" below make it sound like this has a constructor/destructor semantic. but the overall comment says connection established/connection terminated which is much easier to understand. What about renaming "sync" and "unsync" to onConnectionEstablished and onConnectionTerminated (or so) instead? I think it makes the code easier to follow